stable/mongodb/templates/deployment-standalone.yaml

```diff
@@ -44,6 +44,9 @@ spec:
 {{- end }}
       {{- end }}
     spec:
+      {{- if .Values.serviceAccount.create}}
+      serviceAccountName: {{ default (include "mongodb.fullname" .) .Values.serviceAccount.name }}
```

Comment:
Instead of doing this every time you need the serviceAcount name it would be nice if you can add a template to return the name:
```
{{/*
Create the name of the service account to use
*/}}
{{- define "mychart.serviceAccountName" -}}
{{- if .Values.serviceAccount.create -}}
    {{ default (include "mychart.fullname" .) .Values.serviceAccount.name }}
{{- else -}}
    {{ default "default" .Values.serviceAccount.name }}
{{- end -}}
{{- end -}}
```